src/run_jack.rs

```diff
@@ -5,13 +5,16 @@ use jack::*;
 use std::sync::{Arc, Mutex};
 use std::*;
 
-pub fn run_jack_generator(
-    name: String,
-    generator: Arc<Mutex<Generator>>,
-) -> Result<AsyncClient<(), ProcessHandler_>, Error> {
+pub fn make_client(name: String) -> Result<Client, Error> {
```

Comment:
Simplify to one function returning a struct that owns the `AsyncClient` and that allows you to set frequency, etc.

Reply:
Done.